src/EpisodeDataService/CreateEpisode/CreateEpisode.cs

```diff
@@ -152,8 +152,13 @@ private async Task<Episode> MapEpisodeDtoToEpisode(InitialEpisodeDto episodeDto,
         return codeObject;
     }
 
-    private async Task<long> GetOrganisationId(string organisationCode)
+    private async Task<long?> GetOrganisationId(string organisationCode)
     {
+        if (!string.IsNullOrWhiteSpace(organisationCode))
```

Comment:
I don't think we need the not. Do we want to log (a warning or error?) or raise an error here though?

```suggestion
        if (!string.IsNullOrWhiteSpace(organisationCode))
```

Reply:
Oh yes of course, didn't notice that. Thanks Graham!

Reply:
I'm thinking of logging an error